src/test/java/org/broadinstitute/consent/http/resources/DatasetResourceTest.java

```diff
@@ -881,9 +887,9 @@ public void testCreateDatasetRegistration_invalidSchema_case3() {
     }
 
     @Test
-    public void testCreateDatasetRegistration_validSchema() {
+    public void testCreateDatasetRegistration_validSchema() throws SQLException, IOException {
```

Comment:
For tests, its perfectly fine to throw a general exception:
```suggestion
    public void testCreateDatasetRegistration_validSchema() throws Exception {
```